samples/kvsWebRTCClientMaster.c

```diff
@@ -23,24 +23,33 @@ INT32 main(INT32 argc, CHAR *argv[])
     signal(SIGINT, sigintHandler);
 
     // do tricketIce by default
+    printf("[KVS Master] Using trickleICE by default\n");
```

Comment:
Mind doing `printf` -> `PRINTF`. I am not sure what value is added by using the macro, but just to be consistent.

Reply:
Based on offline discussion, keeping it lower case printf since we are turning sample applications to idiomatic C and printf is a C standard function.